src/main/java/net/mcreator/ui/modgui/ProcedureGUI.java

```diff
@@ -403,6 +405,41 @@ public Component getListCellRendererComponent(JList<? extends VariableElement> l
 			}
 		});
 
+		localVarsList.addMouseListener(new MouseAdapter() {
+			@Override public void mouseClicked(MouseEvent e) {
+				if (localVars.getSize() > 0 && e.getClickCount() == 2) {
+					VariableElement selectedVar = localVarsList.getSelectedValue();
+					VariableElementType type = selectedVar.getType();
+					String blockXml = "<xml xmlns=\"http://www.w3.org/1999/xhtml\"><block type=\"variables_"
+							+ (!e.isAltDown() ? "get_" : "set_") + type.getName() + "\"><field name=\"VAR\">local:"

```

Comment:
I would place setter on alt or ctrl down

Reply:
It already does that with alt; I tried using ctrl, but it can also deselects the variable

Reply:
Oh you are right.